test/parallel/test-symlinked-peer-modules.js

```diff
@@ -0,0 +1,78 @@
+// Flags: --preserve-symlinks
+/* eslint no-irregular-whitespace: 0 */
```

Comment:
This isn't supposed to happen. You must have some special Unicode space characters in this file.